uber/models.py

```diff
@@ -1182,14 +1182,14 @@ def last_first(cls):
     @property
     def watchlist_guess(self):
         try:
-            with sa.Session() as session:
-                return session.guess_attendee_watchentry(self)
+            with Session() as session:
+                return [u.__dict__ for u in session.guess_attendee_watchentry(self)]
```

Comment:
Two questions:

1) Why did you use ``u`` as a variable name here?  I'm generally fine with one-character variable names in list comprehensions (I use ``[a for a in attendees ...]`` and [``s for s in staffers ...]`` a lot myself) but I can't figure out what ``u`` is supposed to mean.  (Since they're on the watchlist, maybe it's like you're saying "YOUUUUUUUUU" :P)

2) Why ``.__dict__`` instead of ``.to_dict()``?  The latter is definitely preferable for reasons outlined in a previous PR discussion.